models/repo.go

```diff
@@ -783,7 +783,7 @@ func (repo *Repository) NextIssueIndex() int64 {
 }
 
 var (
-	descPattern = regexp.MustCompile(`https?://\S+`)
+	descPattern = regexp.MustCompile(`https?://[a-zA-Z0-9-_@:;%.,+~#?&/=|!$]+`)
```

Comment:
I don't think, that that is ideal. For example https://www.rügen.de is a valid domain. Also dots, exclamation marks and so on at the end of the url should probably not be included. Just like github doesn't link the punctuation here: http://example.org... or here: http://example.org?

Reply:
Interesting, GitHub doesn't have an URL detection in the description: https://github.com/axifive/test